CR: TLSbo91374
Patch name: TLSbo91374_SDMA_Events_Table_fix.patch
Date: Mon Feb 26 08:58:12 2007 -0600
Source: Freescale Semiconductor, Inc.
Description: Updated the SDMA events table for various platforms
===================================================================
diff -uNpr linux-2.6.19.2.orig/include/asm-arm/arch-mxc/dma.h linux-2.6.19.2/include/asm-arm/arch-mxc/dma.h
--- linux-2.6.19.2.orig/include/asm-arm/arch-mxc/dma.h	2007-04-25 17:37:00.000000000 -0600
+++ linux-2.6.19.2/include/asm-arm/arch-mxc/dma.h	2007-04-25 17:37:57.000000000 -0600
@@ -75,6 +75,8 @@ typedef enum mxc_dma_device {
 	MXC_DMA_CSPI1_TX,
 	MXC_DMA_CSPI2_RX,
 	MXC_DMA_CSPI2_TX,
+	MXC_DMA_CSPI3_RX,
+	MXC_DMA_CSPI3_TX,
 	MXC_DMA_ATA_RX,
 	MXC_DMA_ATA_TX,
 	MXC_DMA_MEMORY,
diff -uNpr linux-2.6.19.2.orig/include/asm-arm/arch-mxc/mx31.h linux-2.6.19.2/include/asm-arm/arch-mxc/mx31.h
--- linux-2.6.19.2.orig/include/asm-arm/arch-mxc/mx31.h	2007-04-25 17:37:51.000000000 -0600
+++ linux-2.6.19.2/include/asm-arm/arch-mxc/mx31.h	2007-04-25 17:37:57.000000000 -0600
@@ -279,7 +279,7 @@
 /*
  * DMA request assignments
  */
-#define DMA_REQ_GPIO1_0    31
+#define DMA_REQ_ECT        31
 #define DMA_REQ_NFC        30
 #define DMA_REQ_SSI1_TX1   29
 #define DMA_REQ_SSI1_RX1   28
@@ -297,10 +297,12 @@
 #define DMA_REQ_FIRI_RX    16
 #define DMA_REQ_UART2_TX   17
 #define DMA_REQ_UART2_RX   16
-#define DMA_REQ_GPIO1_1    15
-#define DMA_REQ_GPIO1_2    14
+#define DMA_REQ_EXTREQ1    15
+#define DMA_REQ_EXTREQ2    14
 #define DMA_REQ_UART4_TX   13
 #define DMA_REQ_UART4_RX   12
+#define DMA_REQ_CSPI3_TX   11
+#define DMA_REQ_CSPI3_RX   10
 #define DMA_REQ_UART5_TX   11
 #define DMA_REQ_UART5_RX   10
 #define DMA_REQ_CSPI1_TX   9
@@ -314,7 +316,7 @@
 #define DMA_REQ_ATA_TX     3
 #define DMA_REQ_ATA_TX_END 2
 #define DMA_REQ_CCM        1
-#define DMA_REQ_reserved   0
+#define DMA_REQ_EXTREQ0    0
 
 /*
  * Interrupt numbers
